src/main/webapp/assets/modules/common/term.ser.js

```diff
@@ -24,7 +24,24 @@ angular.module("Common").factory("termService", function (jsonValue) {
         }
         term.color = fnColor(term.term);//allow d3js select color for unlisted items
       }
+      instance.refineTerm(term);
       return term;
+    },
+
+    refineTerm: function(term) {
+      if (term['averageSalaryMin'] === undefined && term['averageSalaryMax'] === undefined) {
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>['averageSalaryMin'] is better written in dot notation.<br>['averageSalaryMax'] is better written in dot notation.